platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformWiring.java

```diff
@@ -683,6 +689,8 @@ public void bind(
         signedStateHasherWiring.bind(signedStateHasher);
         notifierWiring.bind(notifier);
         platformPublisherWiring.bind(platformPublisher);
+        stateGarbageCollectorWiring.bind(builder.buildStateGarbageCollector());
```

Comment:
looks like an incorrectly resolved merge conflict here

```
stateGarbageCollectorWiring.bind(builder::buildStateGarbageCollector);
```

is being called twice

Reply:
yep, merge conflict artifact.  removed the supplier form.

Reply:
I think @cody-littley may have actually intentionally switched to the supplier form in a recent PR- 

I recommend reverting your change to this line

Reply:
modified to providing supplier.